src/entities/DealTokenSwap.ts

```diff
@@ -420,6 +420,11 @@ export class DealTokenSwap implements IDeal {
    */
   public clauseDiscussions: Map<string, IDealDiscussion>;
 
+  @computedFrom("dealDocument.clauseDiscussions")
+  public get clauseDiscussionsV2(): IDealTokenSwapDocument["clauseDiscussions"] {
```

Comment:
why not just change the declaration of clauseDiscussions?

Reply:
Done.
Changed into getter.